abstractions/typescript/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "incremental": true,                   /* Enable incremental compilation */
     "target": "es2020",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
     "module": "es2020",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
-    "lib": [ "ES2015", "ES2016", "ES2017", "ES2018", "ES2019", "ES2020"],                             /* Specify library files to be included in the compilation. */
+    "lib": [ "ES2015", "ES2016", "ES2017", "ES2018", "ES2019", "ES2020", "dom"],                             /* Specify library files to be included in the compilation. */
```

Comment:
Won't taking a dependency on dom be a problem for backend node projects?

Reply:
No. That config brings in the lib.dom.d.ts.

Reply:
Please don't resolve an item until a common understanding has been reached, it makes it harder to track.

That's my point, I don't believe the node runtime exposes the dom APIs, taking a dependency on those might lead to our libraries using APIs that are not available in this environment. Shouldn't we work only on the smaller common denominator to make sure it runs properly everywhere?